lib/index.js

```diff
@@ -11,5 +12,6 @@ var main = function (options, callback) {
 
 module.exports = main;
 module.exports.launch = main;
+module.exports.argv = argv;
```

Comment:
Currently ember-cli-sauce doesn't implement any kind of arg parsing and just passes everything to saucie: https://github.com/johanneswuerbach/ember-cli-sauce/blob/master/lib/commands/sauce-launch.js

If this is changed, ember-cli-sauce would also need to be changed and two lists of arguments would need to be maintained.